packages/feedback-spa/src/environments/environment.ts

```diff
@@ -7,9 +7,11 @@ export const environment = {
   production: false,
```

Comment:
@anjneeksharma Can you add the `dotenv-plugin` to angular and use that instead of environment.ts file?
That is easier to manage and will make is very easy for deploying to different environments.

Reply:
@deshmukhmayur As per my understanding Angular does not have access to process.env at compile time so cant read it.
The only thing i found in google environement.ts can be created at build time using *magic* file but again it will be same what i already implemented.

Reply:
I think you can customize the build from `angular.json`. But nevermind, I don't know where I was going with that. `environment.ts` is fine.